spring-batch-2/pom.xml

```diff
@@ -48,11 +48,17 @@
             <version>${awaitility.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Is this needed?

Reply:
No, I removed that dependency since hsqldb is not used.